projects/05-fleet-management-api/README.md

```diff
@@ -148,10 +148,10 @@ consultar todas las ubicaciones de un taxi dado el id y una fecha.
 
 ##### Criterios de aceptación
 
-* El _endpoint_ responde para el id del taxi consultado la siguiente
-información: latitud, longitud y fecha y hora.
+* El _endpoint_ responde para el id del taxi y una fecha consultado
+  la siguiente información: latitud, longitud y timestamp (fecha y hora).
```

Comment:
```suggestion
* El _endpoint_ responde con el id del taxi y una fecha mostrando
  la siguiente información: latitud, longitud y timestamp (fecha y hora).
```
I think `mostrar` is better than `consultar` because show the result is the main goal of this requirement vs consult  is related to how to get information from the database.